modules/documents/spec/api/v3/documents/document_representer_rendering_spec.rb

```diff
@@ -49,13 +49,6 @@
 
   subject { representer.to_json }
 
-  before do
-    allow(user)
-      .to receive(:allowed_to?) do |permission, _|
-      permissions.include?(permission)
-    end
-  end
-
```

Comment:
Couldn't find where this spec is mocking its permissions. I see nothing going on there in the shared examples. Is no mocking needed?